main.py

```diff
@@ -15,11 +15,13 @@
     parser.add_argument(
         '-l', '--loglevel', type=str, default='warning', help='logging level', 
         choices=['info', 'debug', 'warning', 'error', 'critical'])
+    parser.add_argument('-r', '--color',
+        default=False, type=lambda x: (str(x).lower() == 'true'),
+        help='Color logs (True/False)')
     args = parser.parse_args()
 
-    loglevel = getattr(logging, args.loglevel.upper(), logging.WARNING)
-    logging.basicConfig(format='%(asctime)s - %(levelname)s - %(message)s', datefmt='%d/%m/%Y %H:%M:%S', level=loglevel)
+    init_logger(args.loglevel.upper(), args.color)
```

Comment:
What about the fybrik-uuid?  It should be passed to the arrow-flight-server from the blueprint controller.